Assets/Vimeo/Scripts/Recorder/EncoderManager.cs

```diff
@@ -64,6 +64,55 @@ public void BeginRecording()
             }
         }
 
+        private void EncoderSetup()
+        {
+            isRecording = true;
+
+#if VIMEO_LOOKING_GLASS_SUPPORT
+            if (_recorder.captureLookingGlassRT) {
+                HoloPlay.Quilt quilt = GetHoloPlayQuilt();
+                if (quilt != null) {
+                    _recorder.renderTextureTarget = quilt.quiltRT;
+
+    #if VIMEO_AVPRO_CAPTURE_SUPPORT
+                    if (_recorder.encoderType == EncoderType.AVProMovieCapture && _recorder.avproEncoder != null) {
+                        RenderHeads.Media.AVProMovieCapture.CaptureFromTexture avproTexture = _recorder.avproEncoder.gameObject.GetComponent<RenderHeads.Media.AVProMovieCapture.CaptureFromTexture>();
+                        if (avproTexture != null) {
+                            avproTexture.SetSourceTexture(_recorder.renderTextureTarget);
+                        }
+                        else {
+                            Debug.LogError("[VimeoRecorder] In order to use AVPro and HoloPlay together, you need to use the CaptureFromTexture component.");
+                        }
+                    }
+    #endif 
```

Comment:
For `define` if statements I think it makes sense to also annotate what each `#endif` closes to make it easier to read

```
#endif //VIMEO_AVPRO_CAPTURE_SUPPORT
```